tensorly/contrib/decomposition/_tt_cross.py

```diff
@@ -426,7 +426,7 @@ def maxvol(A):
         A_new = A_new - A_new * tl.reshape(projection, (tl.shape(A_new)[0], 1))
 
         # Delete the selected row
-        mask.pop(max_row_idx)
+        mask.pop(tl.to_numpy(max_row_idx))
```

Comment:
MXNet does not automatically make scalars standard Python types, and so this is required to work with pop.

Reply:
Yes, I was explicitely typecasting in argmax to make up for that strange behaviour.